ui/app/models/job.js

```diff
@@ -431,7 +431,7 @@ export default class Job extends Model {
 
   @attr('number') version;
 
-  @hasMany('job-versions', { async: true }) versions;
+  @hasMany('job-versions', { async: false }) versions;
```

Comment:
Am I understanding the docs right that this change makes it so that we're always reading whatever data has already been fetched, rather than returning a promise to await to go get that data? And that's ok because we're fetching the data as-needed elsewhere?

Reply:
Your read is exactly correct!

I'm reverting this now to make the /versions fetch explicitly happen because of the route model, but the theory was that since my controller is doing the work of hitting /versions anyway, I can assume that I've already got it. The `watchRelationship` hook would fire on query unblock anyway.

However, after testing in shoddy-internet situations, this results in a bad experience where everything-but-versions on the page could load. Moved it back to async: true in order for the watcher to force resolution before page load.